src/CVODES/CMakeLists.txt

```diff
@@ -0,0 +1,1215 @@
+# ---------------------------------------------------------------
```

Comment:
that's original Cmake script provided by Sundials, so I just used it to compile CVODE static lib for Linux